awx/main/constants.py

```diff
@@ -88,7 +88,8 @@
 
 # :z option tells Podman that two containers share the volume content with r/w
 # :O option tells Podman to mount the directory from the host as a temporary storage using the overlay file system.
+# :ro or :rw option to mount a volume in read-only or read-write mode, respectively. By default, the volumes are mounted read-write.
 # see podman-run manpage for further details
 # /HOST-DIR:/CONTAINER-DIR:OPTIONS
-CONTAINER_VOLUMES_MOUNT_TYPES = ['z', 'O']
+CONTAINER_VOLUMES_MOUNT_TYPES = ['z', 'O', 'ro', 'rw']
```

Comment:
Is there a reason we dont want to support [all of these](https://docs.podman.io/en/latest/markdown/podman-run.1.html#volume-v-source-volume-host-dir-container-dir-options)?

```
              • rw|ro

              • z|Z

              • [r]shared|[r]slave|[r]private[r]unbindable

              • [r]bind

              • [no]exec

              • [no]dev

              • [no]suid

              • [O]

              • [U]
```